### DISCUSSION TITLE
Profile banners

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#14496] [Ideas]
Jist like we will be doing for communities, we should allow users to add a banner to their profile.

We could use the peach colored area in the screenshot to display a banner image, which is how it's displayed in the community view.

![Screenshot_20220604-154624_Brave](https://user-images.githubusercontent.com/65834069/174425072-e9c5b95f-b0c7-4ecf-87f8-a819b746cca5.png)

**Comment by John-44** [comment]
@flexsurfer from a design point of view, we would love to give users the ability to add a banner to the top of their profile.  However @cammellos pointed out that this wasn't technically possible at this time, because it would result in too much load being placed on Waku.  This will of course stop being an issue when we have Codex :-)

@cammellos if this is possible to do from a technical point of view, we would love to add this feature.

**Comment by cammellos** [comment]
It's technically feasible, but at the cost of bandwidth, so it's a tradeoff between higher bandwidth usage and stress to the network over the benefits the feature provides
